data/starts.txt

```diff
@@ -0,0 +1,220 @@
+	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
+	`	You sign your name one final time:`
+	name
+	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
+	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`
+
+
+
+start "hai space"
+	name "Hai Origins"
+	description `You grew up on Greenwater, a world with a rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending locations in human space.`
+	description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
+	thumbnail "scene/hai start"
+	system "Fah Soom"
+	planet "Greenwater"
+	date 16 11 3013
+	to reveal
+		has "Ask the Hai about the Unfettered: offered"
+	on reveal
+		name "Hai Origins"
+		description `You grew up on Greenwater, in one of the human cities built on the equator, where the climate is too hot for the Hai. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending locations in human space.`
+		description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
+		system "Fah Soom"
+		planet "Greenwater"
+		date 16 11 3013
+		credits 1431000
+		debt 1431000
+	to unlock
+		has "main plot completed"
+		has "Ask the Hai about the Unfettered: offered"
+		# TODO: think about what point of HR should unlock this once it's back in the game, if any
+		# has "event: hai-human resolution announced"
+	conversation "hai intro"
+	account
+		credits 1431000
+		score 400
+		mortgage Mortgage
+			principal 1431000
+			interest 0.003
+			term 730
+	set "species: human"
+	set "hai space start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
```

Comment:
`"Intro [0]"` should have its offer conditions changed to not offer if you're not on the default start. Doing it this way breaks `"Default Start Patch"`.
I think you also need the conditions for the other Hai first contact missions offering (which I thought was already done here? I might've accidentally changed it).
```suggestion
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "First Contact: Unfettered: offered"
	set "Ask the Hai about the Unfettered: done"
```

Reply:
it wasn't the case for the unfettered first contact but I guess one could assume you know about them already as part of working in a travel agency